tea.go

```diff
@@ -122,6 +123,10 @@ type Program struct {
 	// as this value only comes into play on Windows, hence the ignore comment
 	// below.
 	windowsStdin *os.File //nolint:golint,structcheck,unused
+
+	// frameRate specifies a custom maximum interval at which we should
+	// update the view. If it is 0, the default value is used.
+	frameRate time.Duration
```

Comment:
I'd also name this var `framerate` (all lowercase) for consistency with the other values.